data/mods/Aftershock/recipes/recipes.json

```diff
@@ -219,6 +219,15 @@
     "using": [ [ "welding_standard", 6 ] ],
     "components": [ [ [ "afs_scrap_titanium", 2 ] ] ]
   },
+  {
+    "result": "afs_titanium_foldable_frame",
+    "type": "recipe",
+    "copy-from": "afs_titanium_frame",
+    "activity_level": "BRISK_EXERCISE",
+    "difficulty": 4,
+    "time": "60m",
+    "components": [ [ [ "afs_scrap_titanium", 2 ], [ "afs_titanium_frame", 1 ] ] ]
```

Comment:
Recipe should take some tools, whatever is used to make vanilla frames would be alright, with a very reduced welding material cost.

Reply:
i used copy-from the non-foldable titanium frame, it appears to be pulling the tools too https://i.imgur.com/hmYGeAF.png bottom one is the folding recipe, top one is vanilla, or is copy-from something to avoid in recipes?